karmagrambot/commands.py

```diff
@@ -1,19 +1,29 @@
 from telegram.ext import CommandHandler
 
-from .analytics import average_message_length
+import analytics
 
 
 def average_length(bot, update):
     message = update.message
     user_id = message.from_user.id
     chat_id = message.chat.id
 
-    average = average_message_length(user_id, chat_id)
+    average = analytics.average_message_length(user_id, chat_id)
     response = f'{average:.3}'
 
     message.reply_text(response)
 
+def karma(bot, update):
+	message = update.message
```

Comment:
These three lines are also not needed. You can access the members in the call to `get_karma`. IMHO it is more legible than assigning three variables just for this.